src/phpDocumentor/Plugin/Twig/Stage/ConfigureTwig.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace phpDocumentor\Plugin\Twig\Stage;
+
+use Twig\Environment;
```

Comment:
Is this the right type? Since the twig writer expects a `twig_environment`?

Reply:
Twig_Environment and Twig\Environment are the same classes :) One is an alias